lib/src/stripe_payment.dart

```diff
@@ -33,11 +33,11 @@ class StripePayment {
   }
 
   /// https://tipsi.github.io/tipsi-stripe/docs/canMakeNativePayPayments.html
-  static Future<bool> canMakeNativePayPayments(List<String> networks) async {
+  static Future<bool> canMakeNativePayPayments(Map<String, dynamic> options) async {
```

Comment:
Could you make this a bit more type-safe?
E.g. keep the `List<String> networks` parameter and send over a map instead, just like below:
`{"networks": networks}`

Reply:
My rationale for `Map<String, dynamic> options` was to mirror the Tipsi project as much as possible.
Their current implementation is here https://github.com/tipsi/tipsi-stripe/blob/1d89c82ac8d5c607bd5563c19ac85a7854be39aa/src/Stripe.js#L66

Reply:
I agree on that but then again it's javascript - all other methods aren't typed as well and that's not really something we'd want in a Dart library :-)

Reply:
Thanks for the fix! Could you change the key to "networks"? That should be the right one.

Reply:
Ah! Apologies for the typo